delta/sdk-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/views/indexing/IndexingStreamBehaviour.scala

```diff
@@ -244,6 +251,10 @@ object IndexingStreamBehaviour {
     */
   private case object ViewIsDeprecated extends IndexingViewCommand[Nothing]
 
+  /** Internal command when the underlying view is invalid and that the entity should be passivated
+    */
+  private case object ViewIsInvalid extends IndexingViewCommand[Nothing]
```

Comment:
Indexing can now fail when starting because the pipeline can't be parsed (missing pipe, invalid conf, ...)
So we passivate like we do when a view is deprecated